### PR TITLE
feat: Adding documentation and fixes to issue templates

### PR DESCRIPTION
**Pull request by 3manuek**
Signed-off-by: 3manuek <emanuel@ongres.com>

### Description

- Added documentation issue template.
- Fixes on templates wording.
 

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+assignees:
+  - gopinathpai
+  - nasbyj
+  - rcv-aws
+  - susanmdouglas-aws
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this documentation improvement!
```
Comment (nasbyj) [inline .github/ISSUE_TEMPLATE/documentation.yml:14]: s/!/./

File: .github/ISSUE_TEMPLATE/enhancement.yml [hunk 2]
```diff
@@ -3,12 +3,14 @@ description: Propose an enhancement for Babelfish for PostgreSQL Website
 title: "[Enhancement]: "
 labels: ["enhancement", "untriaged"]
 assignees:
-  - amazon-auto
+  - 3manuek
+  - gopinathpai
+  - xaviersierra
```
Comment (nasbyj) [inline .github/ISSUE_TEMPLATE/enhancement.yml:8]: Please add me as well. Or maybe we should setup some Github teams and assign stuff to them?

**Review by nasbyj — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+assignees:
+  - gopinathpai
+  - nasbyj
+  - rcv-aws
+  - susanmdouglas-aws
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this documentation improvement.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/documentation.yml:14]: How about:

Thanks for taking the time to fill out this documentation improvement request.

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: Please enter your mail address for contacting in the case of needing more information.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/documentation.yml:19]: How about:

Please provide an email address where we can contact you for more information.

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: documentation-scope
+    attributes:
+      label: What needs to be changed, updated or improve in the Babelfish Documentation?
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/documentation.yml:26]: How about:

What needs to be changed, updated, or improved in the Babelfish documentation?

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+      description: Be explanatory, as detailed as you can. 
+      placeholder: Tell us what you have in mind!
+      value: "This is my feature request!"
+    validations:
+      required: true
+  - type: textarea
+    id: brief-desc-implementation
+    attributes:
+      label: If want to provide us a more details about how to implement.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/documentation.yml:35]: How about:

Provide more details about how to best implement this change.

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+      description: Share with us if you have more details about the implementation.
+    validations:
+      required: false 
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: Select the version of the targeted documentation.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/documentation.yml:43]: How about:

Select the Babelfish documentation version to which the change applies.

File: .github/ISSUE_TEMPLATE/enhancement.yml [hunk 8]
```diff
@@ -3,24 +3,27 @@ description: Propose an enhancement for Babelfish for PostgreSQL Website
...
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Thanks for taking the time to fill out this feature request.
   - type: input
     id: contact
     attributes:
       label: Contact Details
-      description: How can we get in touch with you if we need more info?
+      description: Please enter your mail address for contacting in the case of needing more information.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/enhancement.yml:19]: How about:

Provide an email address where we can contact you if we need more information.

File: .github/ISSUE_TEMPLATE/enhancement.yml [hunk 9]
```diff
@@ -3,24 +3,27 @@ description: Propose an enhancement for Babelfish for PostgreSQL Website
...
       placeholder: ex. email@example.com
     validations:
       required: false
   - type: textarea
     id: what-intends
     attributes:
-      label: What this feature/enhancement tries to solve?
+      label: Describe what this feature intends to solve.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/enhancement.yml:26]: How about:

Describe how this change will improve Babelfish.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 10]
```diff
@@ -3,38 +3,32 @@ description: File a bug report for Babelfish Website
...
     validations:
       required: false
   - type: textarea
     id: what-happened
     attributes:
-      label: What happened?
-      description: Also tell us, what did you expect to happen?
-      placeholder: Tell us what you see!
+      label: Describe the bug
+      description: Describe what was the expected behavior of the website.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/bug.yml:27]: How about:

Describe the expected behavior.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 11]
```diff
@@ -3,38 +3,32 @@ description: File a bug report for Babelfish Website
...
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Thanks for taking the time to fill out this bug report.
   - type: input
     id: contact
     attributes:
       label: Contact Details
-      description: How can we get in touch with you if we need more info?
+      description: Please enter your mail address for contacting in the case of needing more information.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/bug.yml:19]: How about:

Please provide an email address where we can contact you for more information.

File: .github/ISSUE_TEMPLATE/documentation.yml (hunk 4 above)
Comment (3manuek) [reply .github/ISSUE_TEMPLATE/documentation.yml]: We're going to remove all _contact details_ requests in the templates.

File: .github/ISSUE_TEMPLATE/enhancement.yml (hunk 8 above)
Comment (3manuek) [reply .github/ISSUE_TEMPLATE/enhancement.yml]: Removed Contact Details section.

File: .github/ISSUE_TEMPLATE/enhancement.yml (hunk 9 above)
Comment (3manuek) [reply .github/ISSUE_TEMPLATE/enhancement.yml]: I want to emphasize that the _enhacement_ template is for the website specifically. @susanmdouglas-aws  WDYT about:

> Describe how this change will improve Babelfish website.

File: .github/ISSUE_TEMPLATE/enhancement.yml [hunk 12]
```diff
@@ -1,49 +1,44 @@
...
-      placeholder: ex. email@example.com
-    validations:
-      required: false
+        Thank you for taking the time to fill out this feature request.
   - type: textarea
     id: what-intends
     attributes:
-      label: What this feature/enhancement tries to solve?
-      description: Be explanatory, as detailed as you can.
+      label: Describe how this change will improve Babelfish website.
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/enhancement.yml:18]: How about:

Describe how this change will improve the Babelfish website.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 13]
```diff
@@ -3,43 +3,29 @@ description: File a bug report for Babelfish Website
...
   - type: textarea
     id: what-happened
     attributes:
-      label: What happened?
-      description: Also tell us, what did you expect to happen?
-      placeholder: Tell us what you see!
+      label: Describe the bug
+      description: Describe the expected behavior of the website. 
+      placeholder: Tell us what you see.
       value: "A bug happened!"
```
Comment (susanmdouglas-aws) [inline .github/ISSUE_TEMPLATE/bug.yml:21]: Is 'value' displayed onscreen to the user?  If so, how about:

Describe the bug that you've encountered and website behavior.

Comment (3manuek) [reply .github/ISSUE_TEMPLATE/bug.yml:21]: `value` is the example content in the field (the light gray text). Applied the change.

**Review by gopinathpai — APPROVED** [review]

**Review by susanmdouglas-aws — APPROVED** [review]

**Pull request merged**